collection-scripts/gather_windows_node_logs

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="/must-gather"
+WINDOWS_NODE_LOGS=$BASE_COLLECTION_PATH/windows
```

Comment:
Do we want to update https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_service_logs#L13 to say `${BASE_COLLECTION_PATH}/host_service_logs/linux`, and then the referenced line can read `${BASE_COLLECTION_PATH}/host_service_logs/windows` Thus helping with the find-ability and placement consistency of the logs.

Reply:
Yes, having uniformly named directories would benefit both customers and support. We should definitely have this change.

Reply:
I have made this change

Reply:
I think given we have both tests and users expecting to find linux logs in one location today we should leave that unchanged but write out windows logs, which are net new, to a new location. So lets revert the relocation of linux logs.